integrationtests/cli/assets/helm_options_disabled/fleet.yaml

```diff
@@ -0,0 +1,7 @@
+helm:
```

Comment:
While not referencing a Helm chart, this `fleet.yaml` still specifies a `helm` tag. Do we know if, say, specifying a `kustomize` tag with a directory would result in the same behaviour? My understanding is that it should, but that might be something we would like to validate as well.